recipes-devtools/gie/tensorrt-samples_7.1.3-1.bb

```diff
@@ -0,0 +1,74 @@
+DESCRIPTION = "NVIDIA TensorRT Samples for deep learning"
```

Comment:
You need to add `RDEPENDS` because `libnvinfer-samples` contains `python` and `bash` scripts.
Python script `download_pgms.py` is needed to run `/usr/src/tensorrt/bin/sample_algorithm_selector` 

I think that this recipe should go to `external/openembedded-layer/recipes-devtools/gie` because of the `RDEPENDS` list